scripts/toolbox/modules/Feed.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import subprocess
+
+
+def start(cfg, operations):
+    # Feed needs to have a local file as input
+    fileName = "work/feedInput.txt"
+    with open(fileName, "w") as text_file:
```

Comment:
It is generally preferred to stick to either snake case or camel case. This is just a stylistic recommendation.

Reply:
Applied